### PR TITLE
Add support for multiple clients on one host

### PR DESCRIPTION
**Pull request by haslersn**

### CODE REVIEW

**Review by Mr-Pi — APPROVED** [review]

File: tasks/create_client.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- name: Set client facts
+  set_fact:
+    openvpn_client_client_name: "{{ openvpn_client.client_name | default(openvpn_client_default_client_name) }}"
+    openvpn_client_server_name: "{{ openvpn_client.server_name | default(openvpn_client_default_server_name) }}"
+    openvpn_client_up_commands: "{{ openvpn_client.up_commands | default(openvpn_client_default_up_commands) }}"
+    openvpn_client_extra_config: "{{ openvpn_client.extra_config | default(openvpn_client_default_extra_config) }}"
+
+- name: Set client facts
+  set_fact:
+    openvpn_client_config_identifier: "{{ openvpn_client_client_name }}-{{ openvpn_client_server_name }}"
```
Comment (SlothOfAnarchy) [inline tasks/create_client.yml:12]: Couldn't these three `set_fact` tasks be merged?

**Comment by haslersn** [comment]
No

File: tasks/create_client.yml (hunk 1 above)
Comment (haslersn) [reply tasks/create_client.yml:12]: No

Comment (dasJ) [reply tasks/create_client.yml:12]: image

Comment (haslersn) [reply tasks/create_client.yml:12]: Yes

**Pull request merged**